.github/workflows/build.yml

```diff
@@ -12,12 +12,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [17,8]
+        java: [8, 11, 17]
```

Comment:
21

Reply:
i tried that;
https://github.com/l3nnartt/Waterfall/actions/runs/6289342348/job/17075748849

Reply:
The problem here is that `distribution: 'adopt'` isn't working anymore for Java 21, because the movement from AdopOpenJDK to Eclipse Temurin there aren't JDK 21 release distributions of AdoptOpenJDK, ... Because of that the distribution needs to be changed to `temurin`.

Reply:
youre right, thanks!

Reply:
Looks like its not available yet, [zulu](https://github.com/actions/setup-java?tab=readme-ov-file#supported-distributions) already has 21, but we can also wait for temurin ig